lib/generation/generators/middleware/state_management/provider_middleware.dart

```diff
@@ -41,21 +41,21 @@ class ProviderMiddleware extends Middleware {
         var providerWidget = '''
         ChangeNotifierProvider(
           create: (context) =>
-              ${modelName}(), 
+              ${modelName}('${node.name}'), 
           child: LayoutBuilder(
             builder: (context, constraints) {
-              var widget = ${MiddlewareUtils.generateVariableBody(node)};
+              var layout = ${MiddlewareUtils.generateVariableBody(node)};
```

Comment:
Could we benefit from removing the `LayoutBuilder` from the view widget, since we are already inside a `LayoutBuilder`? 🤔